source/time_tables/tests/departure_board_test_data.h

```diff
@@ -39,11 +39,13 @@ struct departure_board_fixture {
     departure_board_fixture() : b("20160101") {
         b.sa("SA1")("S1")("S2");
 
-        b.vj("A").name("A:vj1")("A:s", "08:00"_t)("S1", "09:00"_t)("S2", "10:00"_t)("A:e", "11:00"_t);
+        b.vj("A")
+            .route("A:0", "forward")
```

Comment:
I only added forward and backward into the test data set of, because it seems complicated to add an other **line** with **route** without broken **several tests**...